src/components/organisms/contact-content/contact-content.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { contactImg } from "images/Index";
+import DefaultImage from "components/atoms/image/image";
```

Comment:
```suggestion
import Image from "next/image";
```

Please replace this with next/image.